pkg/scheduler/plugins/elasticquota/core/quota_info.go

```diff
@@ -383,6 +383,7 @@ func NewQuotaTopoNode(quotaInfo *QuotaInfo) *QuotaTopoNode {
 }
 
 func (qtn *QuotaTopoNode) addChildGroupQuotaInfo(childNode *QuotaTopoNode) {
+	childNode.parQuotaTopoNode = qtn
```

Comment:
From the name of this function, the original implementation is OK. On the whole, this function is not very necessary.